pyvista/core/filters/poly_data.py

```diff
@@ -1029,6 +1030,10 @@ def tube(
             Maximum tube radius in terms of a multiple of the minimum
             radius.
 
+        absolute : bool, optional
```

Comment:
Side note: since we're touching this I think it would make sense to update this method's docstring to use the new `optional`/`default` convention described in https://dev.pyvista.org/extras/developer_notes.html#docstrings.

Reply:
Thanks for the pointer @adeak. I agree with you. Please, see the next commit, which uses the new `default` convention for all parameters of the method.